README.md

```diff
@@ -1,2 +1,30 @@
-# example-nextjs-app-router
-An example app built using the DevCycle NextJS SDK
+# DevCycle React SDK Example App (TypeScript)
+
+An example app built using [Next.js](https://nextjs.org/) and the [DevCycle Nest.js SDK](https://docs.devcycle.com/sdk/client-side-sdks/nextjs/)
```

Comment:
`DevCycle Nest.js SDK` should be Next not Nest, right?

Reply:
yep 🤦‍♀️